test/test.js

```diff
@@ -2046,6 +2047,193 @@ function tests(dbName, dbType) {
           ]
         });
       });
-    }); 
-  
+    });
+
+  describe(dbType + ': indexes and queries', function () {
```

Comment:
These tests will fail against CouchDB unless we are using the master branch. We should probably switch to using the master branch in Travis, e.g. via Docker.